specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2018-06-01/PerformanceRecommendations.json

```diff
@@ -465,6 +501,15 @@
         }
       },
       "description": "Recommendation action session operation status."
+    },
+    "CloudError": {
+      "x-ms-external": true,
```

Comment:
Why do you re-define ErrorResponse and set 'x-ms-external'? You could use `v2/types.json#definitions/ErrorResponse` This format is correct.